integreat_cms/api/v3/location_categories.py

```diff
@@ -30,6 +31,10 @@ def transform_location_category(location_category, language_slug):
         else location_category.name,
         "color": location_category.color,
         "icon": location_category.icon,
+        "icon_url": settings.STATIC_URL
+        + "/svg/poi-category-icons/"
+        + location_category.icon
+        + ".svg",
     }
```

Comment:
@PeterNerlich The reason you didn't find an easier solution is because you're asking the wrong questions :laughing: 
Just type in "django get static url in view" in Google and look at the [first result](https://stackoverflow.com/a/17738606) :wink: